### PR TITLE
Fix ConciseView to split PositionMessage using [Environment]::NewLine

### PR DESCRIPTION
**Pull request by SteveL-MSFT**
# PR Summary

The change in #11746 conflicted with #11952 resulted in incorrect splitting of the PositionMessage in the ErrorRecord showing the old NormalView line information.  Since we are retaining #11746, need to fix the formatting to use `[Environment]::NewLine`

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the title (the `WIP` bot will keep its status check at `Pending` while the prefix is present) and remove the prefix when the PR is ready.
- **[Breaking changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)**
    - [x] None
    - **OR**
    - [ ] [Experimental feature(s) needed](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/reference/6/Microsoft.PowerShell.Core/About/about_Experimental_Features.md)
        - [ ] Experimental feature name(s): 
- **User-facing changes**
    - [x] Not Applicable
    - **OR**
    - [ ] [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed: 
- **Testing - New and feature**
    - [ ] N/A or can only be tested interactively
    - **OR**
    - [x] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
- **Tooling**
    - [x] I have considered the user experience from a tooling perspective and don't believe tooling will be impacted.
    - **OR**
    - [ ] I have considered the user experience from a tooling perspective and enumerated concerns in the summary. This may include:
        - Impact on [PowerShell Editor Services](https://github.com/PowerShell/PowerShellEditorServices) which is used in the [PowerShell extension](https://github.com/PowerShell/vscode-powershell) for VSCode (which runs in a different PS Host).
        - Impact on Completions (both in the console and in editors) - one of PowerShell's most powerful features.
        - Impact on [PSScriptAnalyzer](https://github.com/PowerShell/PSScriptAnalyzer) (which provides linting & formatting in the editor extensions).
        - Impact on [EditorSyntax](https://github.com/PowerShell/EditorSyntax) (which provides syntax highlighting with in VSCode, GitHub, and many other editors).

### CODE REVIEW

**Review by adityapatwardhan — APPROVED** [review]

File: src/System.Management.Automation/FormatAndOutput/DefaultFormatters/PowerShellCore_format_ps1xml.cs [hunk 1]
```diff
@@ -1100,8 +1100,7 @@ private static IEnumerable<FormatViewDefinition> ViewsOf_System_Management_Autom
                                                 $offsetInLine = 0
```
Comment (PoshChan) [inline src/System.Management.Automation/FormatAndOutput/DefaultFormatters/PowerShellCore_format_ps1xml.cs]: @SteveL-MSFT, your last commit had 2 failures in `PowerShell-CI-static-analysis`
Verify Markdown Links.Verify links in /home/vsts/work/1/s/ADOPTERS.md.https://shell.azure.com/ should work
```powershell
No such device or address
at <ScriptBlock>, /home/vsts/work/1/s/test/common/markdown/markdown-link.tests.ps1: line 114
114:                                 $null = Invoke-WebRequest -uri $url -RetryIntervalSec 10 -MaximumRetryCount 6

```
Verify Markdown Links.Verify links in /home/vsts/work/1/s/CHANGELOG/README.md.preview.md should work
```powershell
Tool reported URL as unreachable.
at <ScriptBlock>, /home/vsts/work/1/s/test/common/markdown/markdown-link.tests.ps1: line 124
124:                             throw "Tool reported URL as unreachable."

```

**Review by daxian-dbw — APPROVED** [review]

**Comment by daxian-dbw** [comment]
@PoshChan please retry static

**Comment by PoshChan** [comment]
@daxian-dbw, successfully started retry of `PowerShell-CI-static-analysis`

File: src/System.Management.Automation/FormatAndOutput/DefaultFormatters/PowerShellCore_format_ps1xml.cs (hunk 1 above)
Comment (PoshChan) [inline src/System.Management.Automation/FormatAndOutput/DefaultFormatters/PowerShellCore_format_ps1xml.cs]: @SteveL-MSFT, your last commit had 4 failures in `PowerShell-CI-static-analysis`
Verify Markdown Links.Verify links in /home/vsts/work/1/s/ADOPTERS.md.https://shell.azure.com/ should work
```powershell
No such device or address
at <ScriptBlock>, /home/vsts/work/1/s/test/common/markdown/markdown-link.tests.ps1: line 114
114:                                 $null = Invoke-WebRequest -uri $url -RetryIntervalSec 10 -MaximumRetryCount 6

```
Verify Markdown Links.Verify links in /home/vsts/work/1/s/ADOPTERS.md.https://shell.azure.com/ should work
```powershell
No such device or address
at <ScriptBlock>, /home/vsts/work/1/s/test/common/markdown/markdown-link.tests.ps1: line 114
114:                                 $null = Invoke-WebRequest -uri $url -RetryIntervalSec 10 -MaximumRetryCount 6

```
Verify Markdown Links.Verify links in /home/vsts/work/1/s/CHANGELOG/README.md.preview.md should work
```powershell
Tool reported URL as unreachable.
at <ScriptBlock>, /home/vsts/work/1/s/test/common/markdown/markdown-link.tests.ps1: line 124
124:                             throw "Tool reported URL as unreachable."

```
Verify Markdown Links.Verify links in /home/vsts/work/1/s/CHANGELOG/README.md.preview.md should work
```powershell
Tool reported URL as unreachable.
at <ScriptBlock>, /home/vsts/work/1/s/test/common/markdown/markdown-link.tests.ps1: line 124
124:                             throw "Tool reported URL as unreachable."

```

**Comment by daxian-dbw** [comment]
https://shell.azure.com/  should be back already, not sure why it's failing.

For the `preview.md` issue reported in the static CI, it's caused by moving `preview.md` to `7.0.0.md` in [this commit](https://github.com/PowerShell/PowerShell/commit/819585cd129a9df2e82e947c212dc76339ae112d#diff-d3bb3391c79904494c60ee2ac2f33070). @adityapatwardhan can you please fix it?

**Comment by daxian-dbw** [comment]
I will merge this PR as it has nothing to do with the failures in the static analysis CI.

**Pull request merged**